web/src/pill_typeahead.ts

```diff
@@ -36,25 +49,42 @@ export function set_up(
         user_source?: () => User[];
         exclude_bots?: boolean;
         update_func?: () => void;
+        help_on_empty_strings?: boolean;
+        display_pill?: (sub: StreamSubscription) => string;
     },
 ): void {
     if (!opts.user && !opts.user_group && !opts.stream) {
         blueslip.error("Unspecified possible item types");
         return;
     }
-    const include_streams = (query: string): boolean =>
-        opts.stream !== undefined && query.trim().startsWith("#");
+    if (!opts.help_on_empty_strings) {
+        opts.help_on_empty_strings = false;
+    }
```

Comment:
Maybe we can use the ternary operator here.

Reply:
Done.